frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -51,19 +54,27 @@ export function Experiment(): JSX.Element {
         addExperimentGroup,
         updateExperimentGroup,
         removeExperimentGroup,
+        setExperimentInsightType,
```

Comment:
we should probably set this on load as well, and use the value from here, vs going into filters